Cargo.toml

```diff
@@ -19,9 +19,9 @@ rust-version = "1.59"
 default = ["std"]
 std = ["unsigned-varint/std", "alloc"]
 alloc = []
-arb = ["quickcheck", "rand", "arbitrary"]
+arb = ["dep:quickcheck", "dep:rand", "dep:arbitrary"]
 scale-codec = ["dep:parity-scale-codec"]
-serde-codec = ["serde"] # Deprecated, don't use.
+serde-codec = ["dep:serde"] # Deprecated, don't use.
```

Comment:
If you would do `serde` instead of `dep:serde` here, I think you wouldn't need to check within the code whether it's `serde` or `serde-codec`, but you could just always rely that `serde` is set.

Reply:
Good point!